static/pdfjs-dist/LICENSE

```diff
@@ -0,0 +1,177 @@
+
```

Comment:
The pdfjs-dist folder should be empty (or just have a readme or dummy file if required). We don't want to include a copy of the viewer itself as part of this repository (also we might have a licensing mismatch/conflict MIT vs Apcache). You have already specified how to copy it in the README.